### ISSUE TITLE
Please add support for Amiga OS 3.2.2

### ISSUE THREAD

**Issue by GoodSpeed001** [henrikstengaard/hstwb-installer#69]
Please add support for Amiga OS 3.2.2 update.

Regards, keep up with good work :)

**Comment by henrikstengaard** [comment]
Will do in either this week or the next

**Comment by GoodSpeed001** [comment]
Thank you for your feedback. I'll then make some tests when available and report back if find any issues.
Regards

**Comment by chapas** [comment]
A new version of Amiga OS 3.2.2.1 is available, can we have this one available as well?
Thanks for all your work.

**Comment by TommyTomorrow** [comment]
Yeah, HstWB update for 3.2.2.x version would be nice because 3.2.2 update f...d up my OS and now I get Guru Meditation...

**Comment by GoodSpeed001** [comment]
Hi, with the new release 3.2.3, it's possible to add support for it?
